dockers/docker-database/supervisord.conf.j2

```diff
@@ -42,13 +42,13 @@ dependent_startup=true
 {%- set ADDITIONAL_OPTS = '' %}
 {%- endif -%}
 command=/bin/bash -c "{ [[ -s /var/lib/{{ redis_inst }}/dump.rdb ]] || rm -f /var/lib/{{ redis_inst }}/dump.rdb; } && mkdir -p /var/lib/{{ redis_inst }} && exec /usr/bin/redis-server /etc/redis/redis.conf --bind {{ LOOPBACK_IP }} {{ redis_items['hostname'] }} --port {{ redis_items['port'] }} --unixsocket {{ redis_items['unix_socket_path'] }} --pidfile /var/run/redis/{{ redis_inst }}.pid --dir /var/lib/{{ redis_inst }} {{ ADDITIONAL_OPTS }}"
-priority=2
```

Comment:
Changes in this file seems not related to BMP feature. Why changing?

Reply:
Revert this back, we can keep redis and redis_bmp as the same priority.

Reply:
Below change is also not related. could you just revert the whole file?

Reply:
This change is necessary, otherwise multiple db instance will generate invalid format of supervisord.conf without starting new line.

Reply:
revert this bug fix change, and use separate PR instead https://github.com/sonic-net/sonic-buildimage/pull/20545